package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-connector-postgresql",
-  "version": "3.9.1",
+  "version": "3.10.0",
```

Comment:
there's no need for this change. When we publish a new version, the tool will update it for us.

Reply:
FIXED

Reply:
Fixed